test_util/dcos_api_session.py

```diff
@@ -88,7 +88,7 @@ def api_request(self, method, path_extension, *, scheme=None, host=None, query=N
                                    query=query, fragment=fragment, port=port, **kwargs)
 
 
-class DcosApiSession(ARNodeApiClientMixin, ApiClientSession):
+class DcosApiSession(ARNodeApiClientMixin, RetryCommonHttpErrorsMixin, ApiClientSession):
```

Comment:
The order of the multiple inheritance is critical. super() in python essentially walks left to right through that list (please no one else kill me for an over simplification)
https://www.youtube.com/watch?v=EiOglTERPEo